README.md

```diff
@@ -67,6 +67,18 @@ The path to the local node_modules folder can be a path relative to the project
 
 If there is no local installation Atom will use the built-in ESLint in the linter-eslint package itself.
 
+## Common Problems
+
+#### Linter not starting when you open a file?
+
+If the linter is not starting when opening a file, it's likely the atom editor is not recognizing the filetype scope, which then won't select to use
```

Comment:
There are _many_ different reasons why it could be not running, just remove the reference to that and make this a tutorial on how to enable a custom language here.